package.json

```diff
@@ -31,7 +31,7 @@
     "typescript": "~4.0.5"
   },
   "scripts": {
-    "prepare": "lerna run prepare",
+    "prepare": "lerna --concurrency=2 run prepare",
```

Comment:
I think @sunix had issues with concurrency settings so probably it should be moved out if this PR as it's not related